interface/patient_file/summary/demographics_full.php

```diff
@@ -339,14 +341,7 @@ function validate(f) {
  return errMsgs.length < 1;
 }
 
-function submitme() {
- var f = document.forms[0];
- if (validate(f)) {
-  somethingChanged = false; // turn off "are you sure you want to leave"
```

Comment:
Whoops, found a problem here.  You zapped this part.  when submitting the form, somethingChanged must be set to false so the beforeunload event handler does not complain about leaving the page.